### DISCUSSION TITLE
yakuza kiwami 2?

### DISCUSSION THREAD

**Discussion by cheche4000** [Kaplas80/TranslationFramework2#51] [Q&A]
funcionaria esto para traducir el yakuza kiwami 2? me interesa hacer una traducción de ese juego aunque nunca he hecho nada parecido

**Comment by Kaplas80** [comment]
Hola,

para el Kiwami 2 tengo otra aplicación, que espero publicar antes de fin de año: https://github.com/Kaplas80/TF3
